lib/src/builder/execute/optimism.rs

```diff
@@ -69,19 +69,18 @@ impl TxExecStrategy<OptimismTxEssence> for OpTxExecStrategy {
         #[cfg(not(target_os = "zkvm"))]
         {
             use chrono::{TimeZone, Utc};
-            use log::info;
             let dt = Utc
                 .timestamp_opt(block_builder.input.timestamp.try_into().unwrap(), 0)
                 .unwrap();
 
-            info!("Block no. {}", header.number);
-            info!("  EVM spec ID: {:?}", spec_id);
-            info!("  Timestamp: {}", dt);
-            info!("  Transactions: {}", block_builder.input.transactions.len());
-            info!("  Fee Recipient: {:?}", block_builder.input.beneficiary);
-            info!("  Gas limit: {}", block_builder.input.gas_limit);
-            info!("  Base fee per gas: {}", header.base_fee_per_gas);
-            info!("  Extra data: {:?}", block_builder.input.extra_data);
+            trace!("Block no. {}", header.number);
+            trace!("  EVM spec ID: {:?}", spec_id);
+            trace!("  Timestamp: {}", dt);
+            trace!("  Transactions: {}", block_builder.input.transactions.len());
+            trace!("  Fee Recipient: {:?}", block_builder.input.beneficiary);
+            trace!("  Gas limit: {}", block_builder.input.gas_limit);
+            trace!("  Base fee per gas: {}", header.base_fee_per_gas);
+            trace!("  Extra data: {:?}", block_builder.input.extra_data);
```

Comment:
This is much less verbose, so I think it should be debug

Reply:
it gets really spammy and repetitive, especially during composition, which already shows block number and hash

Reply:
That's why I like it being debug and not info, it only comes once per block, the logs on tx level are even more frequent and trace